HW5/Valerie/HW5_Task1.py

```diff
@@ -0,0 +1,11 @@
+even_numbers, odd_numbers, not_divisible_numbers = [], [], []
+for i in range(1,11):
+    if i%2 == 0:
+        even_numbers.append(i)
+    elif i%3 == 0:
+        odd_numbers.append(i)
+    elif i%2 != 0 and i%3 != 0:
+        not_divisible_numbers.append(i)
+print(*even_numbers)
+print(*odd_numbers)
+print(*not_divisible_numbers)
```

Comment:
for example:
for x in range(1, 11):
    if x % 2 == 0:
        print(x, 'is even multiple of 2')
    elif x % 3 == 0:
        print(x, 'is an odd multiple of 3')
    else:
        print(x, 'not divisible by 2 and 3')